### DISCUSSION TITLE
Zoom to Selection feature ?

### DISCUSSION THREAD

**Discussion by Mrnofish** [sumatrapdfreader/sumatrapdf#2445] [Ideas]
I often find myself wishing Sumatra had a zoom to selection feature.

Drawing a box on the screen and have Sumatra adjust the view accordingly is a lot faster and more precise than the current options.

**Comment by GitHubRulesOK** [comment]
hold control key and select an area with mouse then you can print selection, copy selection , zoom towards selection (agreed its not one step) or highlight selection at will.

**Comment by Mrnofish** [comment]
I know about Ctrl + Drag, I'm only getting `Copy Selection` not Print or Zoom, though.

**Reply by GitHubRulesOK** [reply]
they are not shown on context the zoom is applied via + and - 
and printing page range as selection is in the print dialog

**Reply by Mrnofish** [reply]
Zoom is still adjusted manually, i.e. selecting before zooming achieves nothing more than manual zooming making Ctrl+Drag a useless step.

What I'm suggesting is Sumatra automatically picking the correct zoom level and centering the view.
